o_el_iks.API/Program.cs

```diff
@@ -6,6 +8,7 @@
 // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
+builder.Services.AddScoped<ICookieToken, CookieToken>();
```

Comment:
Repozytoria i serwisy zazwyczaj powinny mieć w nazwie Repo/Provider/Service - tutaj sugerowałbym np ITokenProvider. Jest to konwencja nazewnicza która pomaga w zrozumieniu tego na co patrzymy. W programowaniu nazywanie rzeczy jest bardzo ważne bo kod który robi coś źle ale jest dobrze opisany i wiadomo co robi jest lepszy niż ten który działa ale nie wiadomo czemu bo nikt go nie rozumie